install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -80,6 +75,10 @@ data:
   conntrack-gc-interval: {{ .Values.conntrackGCInterval | quote }}
 {{- end }}
 
+{{- if hasKey .Values "disableEnvoyVersionCheck" }}
+  disable-envoy-version-check: {{ .Values.disableEnvoyVersionCheck }}
```

Comment:
@joestringer - think the value here needs to be wrapped in quotes.

Reply:
Ah, thanks for reviewing. Does it cause a helm error or something with the current version?

Any chance you would be willing to submit the PR to fix this up?

Reply:
Sure thing, submitted the following PR; https://github.com/cilium/cilium/pull/13911